app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command):
+    old_file = command.split()[1]
+    new_file = command.split()[2]
+
+    if old_file != new_file:
+        with open(old_file, "r") as f:
```

Comment:
Let's use more descriptive names
```suggestion
        with open(old_file, "r") as source:
```